### PR TITLE
fix(Cash Report)

### LINKED ISSUE #3719: Problem with the cash report
There is a difference when consulting the report of the accounts with the reports of the cashbox, this issue proposes to correct this problem

### PR DESCRIPTION
**Pull request by lomamech**
- Use enterprise currency and Use only enterprise currency in Cash report 
- Add column Value for display the debit or credit convert in enterprise   currency 
- Update Report Separated display the difference between incomes and   expense and the balances
closes #3719

### CODE REVIEW

**Comment by lomamech** [comment]
@jniles it's fixed

**Comment by jniles** [comment]
Hi @lomamech,

Thanks for the signal!  I'll take a look.

You asked me about adding labels to this PR - you can use @kwilu to do it, following [this documentation](https://zulip.readthedocs.io/en/latest/contributing/zulipbot-usage.html).

Like this:

@kwilu add "bug fix"

**Comment by kwilu** [comment]
**ERROR:** Label "bug fix" does not exist and was thus not added to this pull request.

**Comment by jniles** [comment]
... looks like it is case-sensitive:

@kwilu add "Bug Fix"

File: server/controllers/finance/reports/cashReport/index.js [hunk 1]
```diff
@@ -101,8 +101,12 @@ function document(req, res, next) {
       _.merge(context, { cashbox });
 
       // determine the currency rendering
-      params.currency_id = cashbox.currency_id;
-      params.isEnterpriseCurrency = params.currency_id === req.session.enterprise.currency_id;
+      // Update By @lomamech
```
Comment (jniles) [inline server/controllers/finance/reports/cashReport/index.js:104]: 👍

**Review by jniles — APPROVED** [review]
Looks good to me.

bors r+

**Pull request merged**